pkgs/applications/misc/msglint/default.nix

```
@@ -0,0 +1,28 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation {
+  name = "msglint-1.04";
+
+  setSourceRoot = "sourceRoot=`pwd`";
+
+  src = fetchurl {
+    url = "https://tools.ietf.org/tools/msglint/msglint-1.04.tar.gz";
+    sha256 = "1yasj6nijip3z86m526bsssd5vgqrx233azvpz1c0z564wngvzbm";
+  };
+
+  patches = [ ./fixfree.patch ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp msglint $out/bin/msglint
+  '';
+
+  meta = with stdenv.lib; {
+    description  = "RFC 822/MIME/DSN/MDN/Tracking-Status message validator.";
+    homepage     = "https://tools.ietf.org/tools/msglint/";
+    downloadPage = "https://tools.ietf.org/tools/msglint/";
+    maintainers  = with maintainers; [ apeyroux ];
+    license      = licenses.unfree;
```

The shipped `LICENSE` file looks like the [MIT license with an additional legal disclaimer](https://fedoraproject.org/wiki/Licensing:MIT#Old_Style_with_legal_disclaimer) for me.

```suggestion
    license      = licenses.mit;
```